src/ChannelServer/World/Skill.cs

```diff
@@ -80,7 +87,10 @@ public Skill(Character character, int skillId, int level)
 			this.Character = character;
 			this.Id = skillId;
 			this.Level = level;
+			this.OverHeatCount = 0; // So they don't reuse this value just cooldown value?
```

Comment:
Class fields are initialized with their default value automatically, you don't need to do this manually.